test/cli-test.js

```diff
@@ -7,13 +7,31 @@ var cliTxt = path.resolve(_files, 'cli.txt');
 var cliPartialsTxt = path.resolve(_files, 'cli_with_partials.txt');
 var moduleVersion = require('../package').version;
 
-var exec = require('child_process').exec;
+var child_process = require('child_process');
```

Comment:
This import should go with the other Node core imports, under the `path` one. We group imports by:
- node core imports
- 3rd party libraries
- relative (same project) imports